src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        String dateFrom = "01.04.2019";
+        String dateTo = "30.04.2019";
+        String[] names = new String[] {"John", "Andrew", "Kate"};
```

Comment:
Remember about informative names of the variables and methods. The variable 'names' could be more descriptive, like 'employeeNames'.